### DISCUSSION TITLE
What is The 'SI" Unit Measurement System?

### DISCUSSION THREAD

**Discussion by Derpercoaster** [OpenRCT2/OpenRCT2#18709] [Q&A] [answered]
In vanilla RCT, there are only two unit measurement systems: Imperial and metric. I know what these both are, but OpenRCT2 added a third one called 'SI'. What does it stand for, if anything, and what is the difference between it and metric, since they don't seem to be different at all? The answer is probably stupidly obvious, and is just evading me, but I would still like to know.

**Comment by spacek531** [comment, marked as answer]
It stands for ["System International"/"International System of Units"](https://en.wikipedia.org/wiki/International_System_of_Units) which is the base system that other metric units are derived from.
